raiden_contracts/data/gas.json

```diff
@@ -1,30 +1,31 @@
 {
-    "CustomToken.approve": 46068,
+    "CustomToken.approve": 46080,
     "CustomToken.mint": 50646,
     "CustomToken.transfer": 36412,
-    "CustomToken.transferFrom": 28916,
+    "CustomToken.transferFrom": 28922,
     "MonitoringService.claimReward": 45921,
-    "MonitoringService.monitor": 230973,
+    "MonitoringService.monitor": 230947,
     "OneToN.bulkClaim 1 ious": 106352,
-    "OneToN.bulkClaim 6 ious": 161980,
+    "OneToN.bulkClaim 6 ious": 161968,
     "OneToN.claim": 103666,
-    "SecretRegistry.registerSecret": 46285,
+    "SecretRegistry.registerSecret": 46297,
     "SecretRegistry.registerSecretBatch1": 46759,
     "SecretRegistry.registerSecretBatch2": 71519,
     "SecretRegistry.registerSecretBatch3": 96279,
     "ServiceRegistry.deposit": 570886,
     "ServiceRegistry.setURL": 47223,
-    "TokenNetwork DEPLOYMENT": 3689329,
-    "TokenNetwork.closeChannel": 122790,
-    "TokenNetwork.openChannel": 115046,
-    "TokenNetwork.setTotalDeposit": 58468,
-    "TokenNetwork.setTotalWithdraw": 115102,
+    "TokenNetwork DEPLOYMENT": 3735537,
+    "TokenNetwork.closeChannel": 122819,
+    "TokenNetwork.openChannel": 115056,
+    "TokenNetwork.openChannelWithDeposit": 153255,
```

Comment:
153255 < (115056 + 58468 = 173524)

That's 12% less. A nice gas reduction in addition to being twice as fast as doing two transactions.

Reply:
That's 20.269, so nearly the cost of a transaction.